hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/FrameRunner.java

```diff
@@ -173,8 +173,8 @@ private long effectiveGasUsed(final long gasLimit, @NonNull final MessageFrame f
     private void propagateHaltException(MessageFrame frame, ExceptionalHaltReason haltReason) {
         if (haltReason.equals(INSUFFICIENT_CHILD_RECORDS)) {
             setPropagatedCallFailure(frame, HevmPropagatedCallFailure.RESULT_CANNOT_BE_EXTERNALIZED);
-        } else if (haltReason.equals(INVALID_FEE_SUBMITTED)) {
```

Comment:
`INVALID_FEE_SUBMITTED` is still used - e.g. `ExchangeRateSystemContract` and `PrngSystemContract`.  Should it be removed from here?

Reply:
I imagine that this response code still needs to be propagated.